include/IDescriptorSet.h

```diff
@@ -10,7 +10,7 @@
 #include "IDescriptorSetLayout.h"
 #include "IGPUBuffer.h"
 #include "STextureSamplingParams.h"
-#include "IVirtualTexture.h"
+#include "IRenderableVirtualTexture.h"
 
 namespace irr
 {

```

Comment:
`IVulkanPrebaked`is not correct cause descriptor set is mutable after creation (just not after bind)

Also don't use virtual inheritance when not needed

Reply:
this file shouldn't exist anyway

Reply:
IDescriptorSet ?

Reply:
teraz its not mine. Mine is in include/irr/asset

Reply:
sorry, it must be an abomination created by me

Reply:
file nuked yet?

Reply:
y resolved